src/ceph.in

```diff
@@ -342,7 +342,9 @@ def format_help(cmddict, partial=None):
 
         if not cmd['help']:
             continue
-        if cmd['flags'] is not None and ((cmd['flags'] & (FLAG_OBSOLETE | FLAG_DEPRECATED)) != 0):
+        flags = cmd.get('flags')
+        if (flags is not None and
```

Comment:
this explicit check for `flags is not None` is clearly fixing the issue, but this is extremely hard to parse due to the uncommon usage of Python's bitwise operators.

Is it really that necessary to use logical bitwise operators to check if a flag is obsolete or deprecated?